pack/eoe/eoep.json

```diff
@@ -449,6 +597,109 @@
+        "name": "Survey the Area",
+        "pack_code": "eoep",
+        "position": 37,
+        "quantity": 2,
+        "text": "While Survey the Area is in your hand or committed to a skill test, it gains [intellect] icons equal to your [agility], and [agility] icons equal to your [intellect].",
+        "traits": "Practiced. Expert.",
+        "type_code": "skill",
+        "xp": 1
+    },
+    {
+        "code": "08038",
+        "cost": 2,
+        "deck_limit": 2,
+        "faction_code": "seeker",
+        "illustrator": "Anna Christenson",
+        "name": "Medical Texts",
+        "pack_code": "eoep",
+        "position": 38,
+        "quantity": 2,
+        "skill_combat": 1,
+        "skill_intellect": 1,
+        "slot": "Hand",
+        "text": "[action]: Choose an investigator at your location and test [intellect] (2). If you succeed, heal 1 damage from that investigator (2 damage instead if you succeed by 2 or more). If you fail, either exhaust Medical Texts or deal 1 damage to that investigator.",
+        "traits": "Item. Tome.",
+        "type_code": "asset",
+        "xp": 2
+    },
+    {
+        "code": "08039",
+        "cost": 0,
+        "deck_limit": 2,
+        "faction_code": "seeker",
+        "illustrator": "Lindsey Messecar",
+        "name": "Unearth the Ancients",
+        "pack_code": "eoep",
+        "position": 39,
+        "quantity": 2,
+        "skill_agility": 1,
+        "skill_intellect": 2,
+        "text": "<b>Investigate.</b> Choose up to 2 [seeker] assets in your hand. The difficulty of this skill test is equal to the combined cost of the chosen assets. If you succeed, put the chosen assets into play, one at a time. For each [[Relic]] asset put into play by this ability, draw 1 card.",
+        "traits": "Insight.",
+        "type_code": "event",
+        "xp": 2
+    },
+    {
+        "code": "08045",
+        "cost": 4,
+        "deck_limit": 2,
+        "faction_code": "seeker",
+        "illustrator": "Drazenka Kimpel",
+        "is_unique": true,
+        "name": "Prophesiae Profana",
+        "pack_code": "eoep",
+        "position": 45,
+        "quantity": 2,
+        "skill_wild": 2,
+        "slot": "Hand",
+        "subname": "Atlas of the Unknowable",
+        "text": "While you are not at the locus, you get +1 [intellect], +1 [agility], and you may ignore attacks of opportunity.\n[reaction] After Prophesiae Profana enters play: Choose a revealed location. That location is \"the locus\" until Prophesiae Profana leaves play.\n[action]: Move any investigator to the locus.",
+        "traits": "Item. Relic. Tome.",
+        "type_code": "asset",
+        "xp": 5
+    },
+    {
+        "code": "08046",
+        "cost": null,
+        "deck_limit": 1,
+        "faction_code": "rogue",
+        "illustrator": "Juan Martinez Pinilla",
+        "name": "Underworld Support",
+        "pack_code": "eoep",
+        "permanent": true,
+        "position": 46,
+        "quantity": 2,
```

Comment:
Guessing this is probably actually `"quantity": 1`, not that it matters *that* much.